charts/knative-net-istio/Chart.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: v2
+name: knative-net-istio
+description: Installs Net Istio for KNative
+type: application
+version: 1.0.1
+appVersion: "v1.0.0"
+maintainers:
+  - name: caraml-dev
+    email: caraml-dev@caraml.dev
+dependencies:
+  - name: knative-serving-core
+    repository: "file://../knative-serving-core"
+    version: 1.1.0
+    alias: knativeServingCore
+    condition: knativeServingCore.enabled
+  - name: generic-dep-installer
+    repository: "file://../helm-dep-installer"
+    version: 0.0.1
+    alias: istiod
+    condition: istiod.enabled
+  - name: base
+    repository: "https://istio-release.storage.googleapis.com/charts"
+    version: 1.13.3
+    alias: base
+    condition: base.enabled
+  - name: generic-dep-installer
+    repository: "file://../helm-dep-installer"
+    version: 0.0.1
+    alias: istioIngressGateway
+    # NOTE: workaround for similar issue: https://github.com/open-telemetry/opentelemetry-helm-charts/issues/228
+    # `global` is of type object, so enabled condition can be specified in there
+    condition: istioIngressGateway.global.enabled
+  - name: generic-dep-installer
+    repository: "file://../helm-dep-installer"
```

Comment:
should we change these to caraml-dev repo URLs? Once we publish that chart?

Reply:
yes, leaving as file url for now, will add a TODO comment